### PR TITLE
[Interpreter] Add support for quantization to BatchedReduceMin/Max

### PR DESCRIPTION
**Pull request by sgeorgiev255**
Summary:
This PR adds support for quantization to BatchedReduceMin and BatchedReduceMax.

Test Plan:
Tests added to OperatorTest.

### CODE REVIEW

**Review by jfix71 — APPROVED** [review]
Couple small nits, but LGTM!

File: lib/Backends/Interpreter/Interpreter.cpp [hunk 1]
```diff
@@ -80,6 +80,10 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
   switch (NI.getKind()) {
   case Kinded::Kind::BatchedReduceMinNodeKind:
   case Kinded::Kind::BatchedReduceMaxNodeKind:
+    return NI.allInputsAndOutputsHaveSameElemKind(
+        {ElemKind::FloatTy, ElemKind::Float16Ty, ElemKind::BFloat16Ty,
+         ElemKind::Int32ITy, ElemKind::Int64ITy, ElemKind::Int8QTy});
+
```
[the remark this answers is not in the document]
Comment (sgeorgiev255) [reply lib/Backends/Interpreter/Interpreter.cpp]: Done.

File: lib/Graph/Graph.cpp [hunk 2]
```diff
@@ -2066,6 +2074,16 @@ Function::createBatchedReduceMin(llvm::StringRef name, NodeValue batch,
   return addNode(new BatchedReduceMinNode(name, OT, batch, axes));
 }
 
+/// Create a node, performing BatchedReduceMax operation. Output type
+/// matches input \p outTy type.
```
[the remark this answers is not in the document]
Comment (sgeorgiev255) [reply lib/Graph/Graph.cpp]: Removed

**Comment by sgeorgiev255** [comment]
@jfix71 let me know if there's anything else :)

**Review by jfix71 — APPROVED** [review]

**Pull request closed without merging**